specifyweb/frontend/js_src/lib/components/Preferences/UserDefinitions.tsx

```diff
@@ -1079,6 +1086,13 @@ export const userPreferenceDefinitions = {
               },
             ],
           }),
+          isReadOnly: definePref<boolean>({
```

Comment:
The read-only feature is enabled not just for record sets, right?
So why is it in the record sets section in preferences